python/python/glide/async_commands/cluster_commands.py

```diff
@@ -1272,3 +1272,138 @@ async def scan(
             List[Union[ClusterScanCursor, List[bytes]]],
             await self._cluster_scan(cursor, match, count, type),
         )
+
+    async def script_exists(
+        self, sha1s: List[TEncodable], route: Optional[Route] = None
+    ) -> TClusterResponse[List[bool]]:
+        """
+        Check existence of scripts in the script cache by their SHA1 digest.
+
+        See https://valkey.io/commands/script-exists for more details.
+
+        Args:
+            sha1s (List[TEncodable]): List of SHA1 digests of the scripts to check.
+            route (Optional[Route]): The command will be routed to all primaries, unless `route` is provided, in which
+            case the client will route the command to the nodes defined by `route`. Defaults to None.
+
+        Returns:
+            List[bool]: A list of boolean values indicating the existence of each script.
```

Comment:
```suggestion
            TClusterResponse[List[bool]]: A list of boolean values indicating the existence of each script.
            When specifying a route other than a single node, response will be:
            {Address (bytes) : response (List[bool]) , ... } with type of Dict[bytes, List[bool]].
```

Reply:
I believe if we specify to all nodes, the results across different nodes are just aggregated using a logical AND, according to [cluster_routing.rs](https://github.com/amazon-contributing/redis-rs/blob/426bb99d0e2d02a3fcb35c28d8d8dd6beae252ef/redis/src/cluster_routing.rs#L338) in redis.rs, and still returns a list of booleans.